src/actions.ts

```diff
@@ -34,23 +34,32 @@ export const freeActionsNevlas = [
 ];
 
 export const freeActionsNevlasPI = [
-  { cost: "4pw", income: "1o,1c" },
-  { cost: "6pw", income: "2o" }
+  { cost: "2pw", income: "1o,1c" },
+  { cost: "3pw", income: "2o" },
+  { cost: "2pw", income: "1q" },
+  { cost: "2pw", income: "1o" },
+  { cost: "1q", income: "1o" },
+  { cost: "2pw", income: "1k" },
+  { cost: "1pw", income: "2c" },
+  { cost: "1k", income: "1c" },
+  { cost: "1o", income: "1c" },
+  { cost: "1o", income: "1t" },
+  { cost: "1t-a3", income: "1k" }
 ];
 
 export const freeActionsBaltaks = [
   { cost: "1gf", income: "1q" }
 ];
 
 export const boardActions =  {
-  [BoardAction.Power1]: { cost: "7pw", income: ["3k"] },
-  [BoardAction.Power2]: { cost: "5pw", income: ["2step"] },
-  [BoardAction.Power3]: { cost: "4pw", income: ["2o"] },
-  [BoardAction.Power4]: { cost: "4pw", income: ["7c"] },
-  [BoardAction.Power5]: { cost: "4pw", income: ["2k"] },
-  [BoardAction.Power6]: { cost: "3pw", income: ["1step"] },
-  [BoardAction.Power7]: { cost: "3pw", income: ["2t"] },
-  [BoardAction.Qic1]: { cost: "4q", income: ["tech"] },
-  [BoardAction.Qic2]: { cost: "3q", income: [">fed"] },
-  [BoardAction.Qic3]: { cost: "2q", income: ["3vp", "pt > vp"]}
+  [BoardAction.Power1]: { cost: "7pw", discounted: "4pw", income: ["3k"] },
+  [BoardAction.Power2]: { cost: "5pw", discounted: "3pw", income: ["2step"] },
+  [BoardAction.Power3]: { cost: "4pw", discounted: "2pw", income: ["2o"] },
+  [BoardAction.Power4]: { cost: "4pw", discounted: "2pw", income: ["7c"] },
+  [BoardAction.Power5]: { cost: "4pw", discounted: "2pw", income: ["2k"] },
+  [BoardAction.Power6]: { cost: "3pw", discounted: "2pw", income: ["1step"] },
+  [BoardAction.Power7]: { cost: "3pw", discounted: "2pw", income: ["2t"] },
+  [BoardAction.Qic1]: { cost: "4q", discounted: "4q", income: ["tech"] },
```

Comment:
What happens if there's a faction that discounts qic usage (eg -1qic) for board actions?